documentation/asciidoc/topics/proc_creating_routes.adoc

```diff
@@ -1,29 +1,30 @@
 [id='creating_routes-{context}']
 = Creating Routes to Services
-In some cases, such as running local {k8s} clusters, you need to create routes
-to access {brandname} clusters from clients running outside {k8s}.
+For services of type `NodePort`, you might also need to create routes to access
```

Comment:
Creating routes doesn't have anything to do with `NodePort` services. For `Routes` in OpenShift you can directly reference default service. No need to create external one.